pom.xml

```diff
@@ -42,7 +42,7 @@
   <licenses>
     <license>
       <name>Apache License, Version 2.0</name>
-      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <url>http://www.apache.org/licenses/LICENSE-${javax.ws.rs-api.version}.txt</url>
```

Comment:
Totally out of scope of this PR. Previous commits were under ASL2 and I'm happy to keep it as that anyway, but probably a broader discussion with PM. So we're not changing this in this PR - possibly in future following PM/BU input.